roles/dtc/common/templates/ndfc_vrf_lite_ebgp.j2

```diff
@@ -0,0 +1,249 @@
+    {% for switch_redist in switch_item.redistribution %}
+      {% if switch_redist.source == 'direct' and switch_redist.route_map %}
+      redistribute direct route-map {{ switch_redist.route_map }}
+    {% endif %}
+      {% if switch_redist.source == 'static' and switch_redist.route_map %}
+      redistribute static route-map {{ switch_redist.route_map }}
+      {% endif %}
+    {% endfor %}
+!
+  {% endif %}
+  {% elif item.bgp.address_family_ipv6_unicast is defined and item.bgp.address_family_ipv6_unicast %}
+    address-family ipv6 unicast
+    {% if item.bgp.address_family_ipv6_unicast.ebgp_distance and item.bgp.address_family_ipv6_unicast.ibgp_distance and item.bgp.address_family_ipv6_unicast.local_distance %}
+      distance {{ item.bgp.address_family_ipv6_unicast.ebgp_distance }} {{ item.bgp.address_family_ipv6_unicast.ibgp_distance }} {{ item.bgp.address_family_ipv6_unicast.local_distance }}
+    {% endif %}
+    {% if item.bgp.address_family_ipv6_unicast.default_originate is defined and item.bgp.address_family_ipv6_unicast.default_originate %}
+      default-information originate
+    {% endif %}
+    {% if item.bgp.address_family_ipv6_unicast.additional_paths_send is defined and item.bgp.address_family_ipv6_unicast.additional_paths_send %}
+      additional-paths send
+    {% endif %}
+    {% if item.bgp.address_family_ipv6_unicast.additional_paths_receive is defined and item.bgp.address_family_ipv6_unicast.additional_paths_receive %}
+      additional-paths receive
+    {% endif %}
+    {% if item.bgp.address_family_ipv6_unicast.additional_paths_selection_route_map is defined and item.bgp.address_family_ipv6_unicast.additional_paths_selection_route_map %}
+      additional-paths selection route-map {{ item.bgp.address_family_ipv6_unicast.additional_paths_selection_route_map }}
+    {% endif %}
+!
+  {% endif %}
+  {% if switch_item.bgp_peers is defined and switch_item.bgp_peers %}
+  {% for peer in switch_item.bgp_peers %}
+    neighbor {{ peer.address }}
+      remote-as {{ peer.remote_as }}
+    {% if peer.local_as is defined and peer.local_as %}
+      local-as {{ peer.local_as }} no-prepend replace-as
+    {% endif %}
+    {% if peer.description is defined %}
+      description {{ peer.description }}
+    {% endif %}
+    {% if peer.update_source is defined and peer.update_source %}
+      update-source {{ peer.update_source }}
+    {% elif switch_item.loopback_id is defined and switch_item.loopback_id %}
+      update-source Loopback{{ switch_item.loopback_id }}
+    {% endif %}
+    {% if peer.disable_connected_check is defined and peer.disable_connected_check %}
+      disable-connected-check
+    {% endif %}
+    {% if peer.remove_private_as_all is defined and peer.remove_private_as_all %}
+      remove-private-as all
+    {% elif peer.remove_private_as is defined and peer.remove_private_as %}
+      remove-private-as
+    {% endif %}
+      ebgp-multihop {{ peer.ebgp_multihop | default(1) }}
```

Comment:
Recommend using defaults as reference check

Reply:
Updated defaults for the ebgp-multihop.